pom.xml

```diff
@@ -445,27 +453,6 @@
                     </filesets>
                 </configuration>
             </plugin>
-            <!-- Javadoc Plug-in -->
-            <plugin>
-                <groupId>org.apache.maven.plugins</groupId>
-                <artifactId>maven-javadoc-plugin</artifactId>
-                <version>3.0.1</version>
-                <configuration>
-                    <skip>${javadoc.skip}</skip>
-                </configuration>
-                <executions>
-                    <execution>
-                        <id>attach-javadocs</id>
-                        <goals>
-                            <goal>jar</goal>
-                        </goals>
-                        <configuration> <!-- add this to disable checking -->
-                            <additionalparam>-Xdoclint:none</additionalparam>
-                        </configuration>
-                    </execution>
-                </executions>
-            </plugin>
```

Comment:
After deleting this, does `mvn javadoc:javadoc` still works? Users may want to solely generate apidocs themselves.

Reply:
Doesn't work again, needs to run `mvn compile site` instead.

Reply:
could you add a github action to automatically compile the site for each release?